### PR TITLE
DE-113756 created sonarqube properties file for integration

### PR DESCRIPTION
**Pull request by ni3nice**

### CODE REVIEW

**Review by akasharya-nice — COMMENTED** [review]
For now, let's remove yml files and only add properties file for creating project in sonar.

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+sonar.projectKey=com.nicedfo.datetime
+sonar.projectName=datetime
+sonar.language=PHP
+sonar.sources=src
+sonar.tests=src
+sonar.exclusions=**/*Test.php
+sonar.test.inclusions=**/*Test.php
+sonar.php.coverage.reportPaths=coverage.xml
+sonar.php.tests.reportPath=test-report.xml
```
Comment (akasharya-nice) [inline sonar-project.properties:9]: for new let's remove coverage part.

Comment (ni3nice) [reply sonar-project.properties]: removed coverage part 
i.e. "sonar.php.coverage.reportPaths=coverage.xml"

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
+sonar.projectKey=com.nicedfo.datetime
+sonar.projectName=datetime
+sonar.language=PHP
+sonar.sources=src
+sonar.tests=src
+sonar.exclusions=**/*Test.php
+sonar.test.inclusions=**/*Test.php
+sonar.php.tests.reportPath=test-report.xml
```
Comment (akasharya-nice) [inline sonar-project.properties:8]: this also needs to go.

Comment (ni3nice) [reply sonar-project.properties]: removed unnecessary code

**Pull request closed without merging**